queries/places.jinja2

```diff
@@ -49,17 +53,17 @@ WHERE
         {% if zoom >= 8 %},  'city', 'town'{% endif %}
         {% if zoom >= 10 %}, 'borough', 'suburb', 'quarter'{% endif %}
         {% if zoom >= 11 %}, 'village'{% endif %}
-        {% if zoom >= 12 %}, 'hamlet', 'neighbourhood'{% endif %}
+        {% if zoom >= 12 %}, 'hamlet'{% endif %}
         {% if zoom >= 13 %}, 'locality', 'isolated_dwelling', 'farm'{% endif %}
     )
     AND {{ bounds|bbox_filter('way') }}
 
-{% if zoom >= 12 %}
+{% if zoom >= 11 %}
 UNION ALL
 
 SELECT
     name,
-    'neighbourhood' AS kind,
+    wnp.placetype_string AS kind,
```

Comment:
I was just taking the prefixes from the table name, ie `w`of_`n`eighbourhood_`p`lacetype. In this case I could have just used `placetype_string` because there were no collisions, but it was the only column from the other table so I figured that this might call it out.

Should that prefix be dropped? Updated to be something else? Or just use the whole other table name?